### PR TITLE
Choreographer v2.1 documentation

### PR DESCRIPTION
**Pull request by szkris**
Added documentation for Choreographer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6660,4 +6666,411 @@ __Ping Measurment response by system id__ the output :
...
+
+<a name="choreographer" />
+ 
+# Choreographer
+ 
+<a name="choreographer_sdd" />
+
+## System Design Description Overview
+
+This supporting core system makes it possible to execute pre-defined workflows through orchestration and consumption.
```
Comment (tsvetlin) [inline README.md]: through orchestration and **service** consumption.

File: README.md [hunk 2]
```diff
@@ -6660,4 +6666,411 @@ __Ping Measurment response by system id__ the output :
...
+
+Each workflow can be divided into three segments: plans, actions and steps. Plans define the whole workflow by name and they contain Actions which group coherent Steps together for greater transparency and enabling sequentialization of these step groups.
```
Comment (tsvetlin) [inline README.md]: Make it a bulletpointed list
Each workflow can be divided into three segments:
* Plans
* Segments
* Steps

In the description use consistent capitalization. I'd suggest, _lowercase italic_.

File: README.md [hunk 3]
```diff
@@ -6660,4 +6666,411 @@ __Ping Measurment response by system id__ the output :
...
+
+Workflow execution in this generation can only be accomplished if the requested providers in each step are all available (they are registered with the same name in the service registry as in the plan description) and the requested services call back (notify) to the Choreographer through the Choreography service that the execution on their end is done. Only this way can the Choreographer continue the execution of the plan.
+
+<a name="choreographer_usecases" />
+
+## Services and Use Cases
+
+This System provides the Choreogher Service which only has one use-case scenario: notifying the Choreographer from the providers' side that the executed step is done.
```
Comment (tsvetlin) [inline README.md]: This Supporting Core System

Choreogher -> Choreographer

File: README.md [hunk 4]
```diff
@@ -6660,4 +6666,411 @@ __Ping Measurment response by system id__ the output :
...
+| Function | URL subpath | Method | Input | Output |
+| -------- | ----------- | ------ | ----- | ------ |
+| [Echo](#choreographer_endpoints_get_echo) | /echo | GET | - | OK |
+| [Notify that a step is done](#choreographer_endpoints_post_nofity) | /notifyStepDone | POST | [SessionRunningStepData](#datastructures_choreographer_session_running_step_data) | OK |
+
+<a name="choreographer_endpoints_mgmt" />
+
+###  Management endpoint description<br />
+
+There endpoints are mainly used by the Management Tool and Cloud Administrators.
```
Comment (tsvetlin) [inline README.md]: There -> These

File: README.md [hunk 5]
```diff
@@ -6660,4 +6666,411 @@ __Ping Measurment response by system id__ the output :
...
+
+| Function | URL subpath | Method | Input | Output |
+| -------- | ----------- | ------ | ----- | ------ |
+| [Get all plan entries](#choreographer_endpoints_get_mgmt_plan) | /mgmt/plan | GET | - | [ChoreographerPlanEntryList](#datastructures_choreographerplanentrylist) |
+| [Add a plan entry](#choreographer_endpoints_post_mgmt_plan) | /mgmt/plan | POST | [ChoreographerPlanEntry](#datastructures_choreographer_addplanentry) | CREATED |
+| [Get a plan entry by ID](#choreographer_endpoints_get_mgmt_plan_id) | /mgmt/plan/{id} | GET | ChoreographerPlanID | [ChoreographerPlanEntry](#datastructures_choreographerplanentry)
```
Comment (tsvetlin) [inline README.md:6724]: Broken link!
[ChoreographerPlanEntry](#datastructures_choreographerplanentry) -> [ChoreographerPlanEntry](#datastructures_choreographer_addplanentry)

File: README.md [hunk 6]
```diff
@@ -6660,4 +6666,411 @@ __Ping Measurment response by system id__ the output :
...
+> **Note:**  In the current version the `parameters`, the `metadata` and the `quantity` fields cannot be used when executing plans. These will be implemented in future versions.
+
+<a name="choreographer_endpoints_post_mgmt_plan" />
+
+### Add a plan entry
+```
+POST choreographer/mgmt/plan
+```
+
+Creates a plan record and returns HTTP 201 - CREATED if the addition is successful or the proper error message if the addition failed.
```
Comment (tsvetlin) [inline README.md]: addition -> entry creation

sounds better in both places

**Review by tsvetlin — CHANGES_REQUESTED** [review]
Please apply the commented changes. After I'll merge it.
Thank you!

**Comment by szkris** [comment]
Applied all changes, PR is ready.

**Review by tsvetlin — APPROVED** [review]
LGTM!

**Pull request merged**